core/domain/user_services.py

```diff
@@ -471,6 +481,19 @@ def get_user_settings(user_id, strict=False):
     return user_settings
 
 
+def get_user_role_from_id(user_id):
+    """Returns the role of the user with given user_id.
+
+    Args:
+        user_id: str. User's id.
```

Comment:
updated to `user_id: str. The user id.` 

It is used in this way in other functions as well.